sentry-rails/lib/sentry/rails/configuration.rb

```diff
@@ -159,6 +159,25 @@ class Configuration
       # Set this option to true if you want Sentry to capture each retry failure
       attr_accessor :active_job_report_on_retry_error
 
+      # Configuration for structured logging feature
+      # @return [StructuredLoggingConfiguration]
+      attr_reader :structured_logging
+
+      # Allow setting structured_logging as a boolean for convenience
+      # @param value [Boolean, StructuredLoggingConfiguration]
+      def structured_logging=(value)
```

Comment:
let's not have the boolean overloading, having the boolean `structured_logging.enabled` inside is enough